examples/run_examples.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Script used to run all examples at once.
+# Does not parse examples automatically. The exact examples that are run
+# are defined here in the script. Run this script from inside the `examples`
+# folder.
+#
+# USAGE:
```

Comment:
I would implement a `-h` argument that returns the usage :-)

Reply:
:dark_sunglasses: